crane/views/v2.py

```diff
@@ -125,3 +127,12 @@ def handle_error(error):
     response.headers['Content-Type'] = 'application/json'
     response.status_code = error.status_code
     return response
+
+
+def get_accept_headers(request):
+    accept_headers = request.headers.get('Accept')
+    if not accept_headers:
+        return []
+    log.debug("Accept: %s", accept_headers)
+    accept_headers = accept_headers.split(',')
+    return set(x.strip() for x in accept_headers)
```

Comment:
for header in accept_header i find easier to read, in general i would avoid x,y,i, etc variable names.

Reply:
I would agree, but for comprehension I find it concise enough. If you care deeply I can change it.

Reply:
not really. it's just my personal preference of the pattern.